### PR TITLE
Update Jupytyerhub kfdef manifest version to 1.0.1

### PR DESCRIPTION
**Pull request by anishasthana**
The 1.0.1 release updates the jupyterhub image so we no longer need to
skip cert validation. More context at https://github.com/opendatahub-io/odh-manifests/pull/312

Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

**Comment by anishasthana** [comment]
Upstream is planning to release 1.0.2 which will hopefully incorporate https://github.com/opendatahub-io/odh-manifests/pull/330. This means we can clean up our metrics endpoint for jupyterhub too.  I'll place it on hold until the release is out so we can confirm.

**Comment by anishasthana** [comment]
/unhold

The metrics fix we want will _not_ be included in the 1.0.2 release, but it will be in 1.0.3

**Comment by 4n4nd** [comment]
let me test this

**Review by tumido — COMMENTED** [review]
/lgtm

/hold for @4n4nd

**Comment by 4n4nd** [comment]
Tested :+1: 
I was able to spawn a server and print hello world in the notebook. No cert issues in the staging namespace.

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Comment by 4n4nd** [comment]
/unhold
After this is merged, we will need to roll out a new deployment for JH.

**Pull request merged**
